test/functional/update/slow-server/test.bats

```diff
@@ -8,6 +8,12 @@ port=""
 targetfile="6c27df6efcd6fc401ff1bc67c970b83eef115f6473db4fb9d57e5de317eba96e"
 
 setup() {
+  # Skip this test if not running in Travis CI, because test takes too long for
```

Comment:
the idea is that the old tests are removed when the new tests (v2) are fully integrated, so I guess it is fins to update this old test here for now, but we should be using the new library for tests instead.

Reply:
I tried to port this test this morning because I thought I had all features I needed, but I think there's still some things to get in before porting it. If you don't mind, I prefer waiting until some updates tests are merged before porting this one. But it's on my radar.

Reply:
perfect